src/pages/patient/CreateCase/CreateCase.js

```diff
@@ -138,6 +139,48 @@ class CreateCase extends Component {
 
 
       </Layout>
+      <div>
+        <h2>Create Case</h2>
+
+        <div>
+          <DatePicker value={illnessStartDate} onChange={this.handleDateChange} />
+
+          <FormField type="textarea" label="symptoms" {...control(this, 'symptoms')} />
+
+          <FormField type="textarea" label="note" {...control(this, 'note')} />
+
```

Comment:
Class has unnecessary blank spaces. This should be corrrected.

Reply:
@KurtTaylan  so sorry :) create-case not class